wikidataQuery.py

```diff
@@ -44,6 +43,14 @@ def makeQuery(keywords):
         answer = submitQuery(entity_id, property_id)
     elif query_type == 'yes/no':
         answer = submitCheckQuery(entity_id, property_id, prop_attribute_id)
+    # TODO make query for each type
+    elif query_type == 'date':
+        answer = submitDateQuery(entity_id, properties_id)
+        
+    elif query_type == 'place':
+        answer = submitPlaceQuery(entity_id, properties_id)
+    #elif query_type == 'cause':
```

Comment:
In case you're not going to implement these two query types in this PR, please list them as TODO.